src/SqlSquared/Signature.purs

```diff
@@ -138,8 +142,20 @@ data SqlF literal a
   | Select (SelectR a)
   | Parens a
 
+type AnyDirPath = E.Either (Pt.AbsDir Pt.Unsandboxed) (Pt.RelDir Pt.Unsandboxed)
```

Comment:
Might be worthing to introduce the same type and print/parse for filepath. It's used in table relation.